Code/GraphMol/Chirality.cpp

```diff
@@ -153,6 +153,46 @@ void controllingBondFromAtom(const ROMol &mol,
   }
 }
 
+namespace {
+void set_stereoany_from_squigglebond(ROMol &mol, Bond *bond) {
```

Comment:
a trivial comment before I do a real review (that's for tomorrow): we don't use snake_case in the RDKit. Please switch this to
```suggestion
void setStereoAnyFromSquiggleBond(ROMol &mol, Bond *bond) {
```

Reply:
I moved this into the detail namespace and also used it in CXSmilesOps to remove duplicated code.